src/NLog.StructuredLogging.Json.Tests/LoggerExtensionsScopeTests.cs

```diff
@@ -54,6 +54,21 @@ public void TearDown()
         }
 
         [Test]
+        public void Should_Fail_If_Logger_Is_Null()
+        {
+            ILogger nullLogger = null;
+            var exeption = Assert.Throws<ArgumentNullException>(() => nullLogger.BeginScope("should fail"));
```

Comment:
Not exactly, it accepts expected message as argument, so fro my opinion it is better to verify parameter name instead of message.